### PR TITLE
feat: Approval data return a dictionary instead of array of hashes

### PR DESCRIPTION
**Pull request by juanmahidalgo**

### CODE REVIEW

File: src/Item/Item.model.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class Item extends Model<ItemAttributes> {
         FROM ${raw(this.tableName)} items
         JOIN ${raw(
           ItemCuration.tableName
-        )} item_curations ON items.id = item_curations.item_id AND
+        )} item_curations ON items.id = item_curations.item_id
```
Comment (LautaroPetaccio) [inline src/Item/Item.model.ts:23]: I was thinking that the tree to be built requires not only this data, but the data of all the items that are published for a third party. We need all the data to re-build the tree, and the tree is built with all the items of a third party.

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

File: src/Item/Item.model.ts (hunk 1 above)
Comment (nicosantangelo) [reply src/Item/Item.model.ts:23]: True, I think then that the query should not check for curation status but rather for curation "existence", which is already being done by the INNER JOIN right?

**Review by nicosantangelo — APPROVED** [review]
The changes are 💯 
The conversation [here](https://github.com/decentraland/builder-server/pull/437/files#r827306650) can be done in another issue

File: src/Item/Item.model.ts (hunk 1 above)
Comment (juanmahidalgo) [reply src/Item/Item.model.ts:23]: Alright, as Nico suggested, we can take care of that in another PR. Issue created: https://github.com/decentraland/builder-server/issues/439

**Pull request merged**